CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 ## 5.4.0 (Unreleased)
 
+DEPRECATIONS:
+* bigquery: deprecated `cloud_spanner.use_serverless_analytics` on `google_bigquery_connection`. Use `cloud_spanner.use_data_boost` instead. ([#6539](https://github.com/hashicorp/terraform-provider-google-beta/pull/6539))
+
+NOTES:
+* provider: added `universe_domain` attribute as a provider attribute ([#6551](https://github.com/hashicorp/terraform-provider-google-beta/pull/6551))
+
+BREAKING CHANGES:
+* cloudrunv2: marked `location` field as required in resource `google_cloud_run_v2_job`. Any configuration without `location` specified will cause resource creation fail ([#6540](https://github.com/hashicorp/terraform-provider-google-beta/pull/6540))
+* cloudrunv2: marked `location` field as required in resource `google_cloud_run_v2_service`. Any configuration without `location` specified will cause resource creation fail ([#6540](https://github.com/hashicorp/terraform-provider-google-beta/pull/6540))
+
+FEATURES:
+* **New Data Source:** `google_cloud_identity_group_lookup` ([#6530](https://github.com/hashicorp/terraform-provider-google-beta/pull/6530))
+* **New Resource:** `google_network_connectivity_policy_based_route` ([#6552](https://github.com/hashicorp/terraform-provider-google-beta/pull/6552))
+* **New Resource:** `google_pubsub_schema_iam_*` ([#6533](https://github.com/hashicorp/terraform-provider-google-beta/pull/6533))
+
+IMPROVEMENTS:
+* accesscontextmanager: added support for Access Context Manager's VPC Sub-segmentation feature ([#6553](https://github.com/hashicorp/terraform-provider-google-beta/pull/6553))
```

Comment:
I think it might make sense to specify the fields that were added rather than the feature?